### DISCUSSION TITLE
How to publish an unpackaged Uno app

### DISCUSSION THREAD

**Discussion by mobynet1** [unoplatform/uno#17761] [Q&A]
I have been struggling with Microsoft and the Maui team for eight months now trying to create an unpackaged app. When you follow the Maui team's instructions, the app is published to the correct output folder, but when the application is run select icons such as font icons do not show up. You only get a placeholder rectangle where the icon should be.

I see in your publish instructions how to publish a packaged app, but this will not work for my application. Our application will be self-distributed and will never be deployed via any of the app stores. Plus, the way the MSIX apps get installed, they are installed in the user's appdata folder so if more than one person uses the computer, you get multiple copies of the app. Not good!

We have always distributed our Windows-based app via an installer generated with the WiX Toolset. My question is: if I just bundle up the contents of the bin\Debug\net8.0-windows10.0.19041, will I suffer the same fate with Uno? I was told by the Maui Product Manager that they suspected the resizetizer as the culprit, and that they had the issue fixed. They did not have it fixed as I verified over this past weekend. Hence, this is driving my decision to drop Maui and move to Uno. I think you guys use your own resizetizer. Is your version **YOUR** version or is it just Microsoft's version rebundled?

**Comment by jeromelaban** [comment]
> When you follow the Maui team's instructions, the app is published to the correct output folder, but when the application is run select icons such as font icons do not show up. You only get a placeholder rectangle where the icon should be.

This is the part where WinAppSDK may be a bit difficult to work with, let us know which scenarios when you find them, but if you find yourself in that situation, you can also use `net8.0-desktop` target framework that also runs properly on Windows.

> I see in your publish instructions how to publish a packaged app, but this will not work for my application. Our application will be self-distributed and will never be deployed via any of the app stores. Plus, the way the MSIX apps get installed, they are installed in the user's appdata folder so if more than one person uses the computer, you get multiple copies of the app. Not good!

Indeed, we do not have the section on how to deploy unpackaged apps, but let's change this with https://github.com/unoplatform/uno/pull/17762.

> We have always distributed our Windows-based app via an installer generated with the WiX Toolset. 

That should work, but same answer as above, you can use the `net8.0-desktop` target if the WinAppSDK tooling is causing trouble.

> My question is: if I just bundle up the contents of the bin\Debug\net8.0-windows10.0.19041, will I suffer the same fate with Uno? I was told by the Maui Product Manager that they suspected the resizetizer as the culprit, and that they had the issue fixed. They did not have it fixed as I verified over this past weekend. Hence, this is driving my decision to drop Maui and move to Uno. I think you guys use your own resizetizer. Is your version YOUR version or is it just Microsoft's version rebundled?

Uno Resizetizer is based on MAUI's version, with additional fixes and additional platforms (wasm, desktop, etc...). Try it out and let us know if there are scenarios that don't work.

**Reply by mobynet1** [reply]
I got the resizetizer to work and I got all of the image transforms done. As soon as I get my Maui app fully converted, I will follow up with how well it went. I am truly liking what I am seeing with Uno. I have made the decision to ditch Maui for Uno. So... Now I'm committed to make it work!

Thanks for your support @jeromelaban! It is greatly appreciated!

**Reply by jeromelaban** [reply]
Thank you for the update! Glad to see you on board!
